### ISSUE TITLE
Style: hiding of output

### ISSUE THREAD

**Issue by stefanv** [scipy/scipy-sphinx-theme-v2#45]
It feels like the hiding of output cells would look better if it hid completely, instead of just making the output disappear.  Currently, you are left with code blocks with large spaces in between.  Thoughts?

**Comment by fuglede** [comment]
Slight disagree: The nice thing about just hiding the outputs is that the vertical locations of the input cells are invariant. This, in turn, has the advantage that when you want to just copy-paste a subset of all the code, you know exactly where that code is after hiding the output (whereas if it jumped around, you'd have to re-find the code you were trying to copy).

That said, if there are examples with truly huge outputs, I can see how that can also be disruptive.

**Comment by stefanv** [comment]
I think my eye would have been less confused if *only* the outputs got touched, but the input prompt also gets removed.  Maybe we can keep the `>>>` but not make those highlight during selection of code.

**Comment by fuglede** [comment]
Good point. One downside would be that it wouldn't be immediately obvious that that's how it works (cf. principle of least astonishment). This would only really be bad if it keeps someone from copy-pasting in the first place because they think "Ah, I don't want to copy-paste because I'd then have to manually remove the `>>>`) without realizing that they wouldn't have to.
